src/scripts/main.js

```diff
@@ -1 +1,102 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+firstPromise
+  .then((message) => {
+    const div = document.createElement('div');
+
+    div.classList.add('success');
+    div.setAttribute('data-qa', 'notification');
+    div.textContent = message;
+
+    document.body.append(div);
+  })
+  .catch((error) => {
+    const div = document.createElement('div');
+
+    div.classList.add('error');
+    div.setAttribute('data-qa', 'notification');
+    div.textContent = error.message;
+
+    document.body.append(div);
+  });
+
+let clickLeft = false;
+let clickRigth = false;
```

Comment:
There is a typo in the variable name 'clickRigth'. It should be 'clickRight' to correctly represent the right click.